Update odo debugging.

boczkowska opened the pull request:
Branch 4.6 and 4.7

bobfuru commented on modules/developer-cli-odo-debugging-an-application.adoc:
```diff
@@ -10,11 +10,83 @@ You can debug your application on in `odo` with the `odo debug` command.
+ ✓  Changes successfully pushed to component
+----
++
+NOTE: You can specify a custom debug command by using the `--debug-command="custom-step"` flag.
```
In the preview build I generated, the `NOTE` admonitions look fine, but our [guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#admonitions) are to format as such:

```
[IMPORTANT]
====
lorem ipsum
====
```
So it might be worth an update to these instances.

bobfuru commented on modules/developer-cli-odo-debugging-an-application.adoc:
```diff
@@ -10,11 +10,83 @@ You can debug your application on in `odo` with the `odo debug` command.
+----
 
-. Attach the debugger bundled in your IDE to the component. Instructions vary depending on your IDE.
+. Attach the debugger bundled in your IDE of choice. Instructions vary depending on your IDE, for example link:https://code.visualstudio.com/docs/nodejs/nodejs-debugging#_remote-debugging[VSCode debugging interface].
```
For clarity: 

- s/debugger bundled/debugger that is bundled
- s/for example/for example:

bobfuru reviewed (commented):
I have a few updates and suggestions.

bobfuru commented:
> The preview will be available shortly at:
> 
> * _modules/developer-cli-odo-debugging-an-application.adoc_: https://updateOdoDebugging--ocpdocs.netlify.com/openshift-enterprise/latest/modules/developer-cli-odo-debugging-an-application.html

The preview build didn't work because it was a single module, so it would be good to include a local build hosted on an internal server. I built one locally to check in this case.

boczkowska commented:
@bobfuru thank you for the review! I've addressed all of the issues. 
Apologies for the absence of the preview, I didn't notice. :( To be fair, it worked when I clicked "Back to our site" and it took me to the preview of the main page from which I could access the module itself - https://updateododebugging--ocpdocs.netlify.app/openshift-enterprise/latest/cli_reference/developer_cli_odo/creating_and_deploying_applications_with_odo/debugging-applications-in-odo.html
Could this be a problem of just the URL generated by the bot? In any case, I'll make sure to check the build next time.

bobfuru commented:
> @bobfuru thank you for the review! I've addressed all of the issues.
> Apologies for the absence of the preview, I didn't notice. :( To be fair, it worked when I clicked "Back to our site" and it took me to the preview of the main page from which I could access the module itself - https://updateododebugging--ocpdocs.netlify.app/openshift-enterprise/latest/cli_reference/developer_cli_odo/creating_and_deploying_applications_with_odo/debugging-applications-in-odo.html
> Could this be a problem of just the URL generated by the bot? In any case, I'll make sure to check the build next time.

Interesting, the link you shared here works for me as well. I should have tried the "back to our site" link before. 😁  I thought we had a limitation with netlify that it wouldn't allow preview builds for single-file modules because they lived outside the context of an assembly. But that seems to not quite be the case ¯\_(ツ)_/¯ @vikram-redhat WDYT?

Changes all LGTM, thanks for the updates. I'll merge/CP to 4.6, 4.7.

bobfuru commented:
/cherrypick enterprise-4.6

bobfuru commented:
/cherrypick enterprise-4.7
